template/.github/workflows/ci.yml

```diff
@@ -32,19 +34,22 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v4
 
+    - name: Install managment tools
+      run: pipx install poetry && pipx install invoke
```

Comment:
Question - Do we need to ensure that pipx is installed and available in the CI env?

Reply:
Apparently not - it is installed by default. This approach is how the `setup-python` actions recommends using Poetry.